### DISCUSSION TITLE
热力图问题

### DISCUSSION THREAD

**Discussion by tiansQAQ** [antvis/G2Plot#3633] [Q&A]
<img width="1258" alt="image" src="https://github.com/antvis/G2Plot/assets/46640919/146e8ab6-74fe-4497-88d2-47319b3c4d36">

<img width="929" alt="image" src="https://github.com/antvis/G2Plot/assets/46640919/ddc64997-d8e6-4a98-8065-5f4c22ac86ba">

怎么才能像第一张图片一样x轴和y轴上加上线条？

**Comment by hustcc** [comment]
搜一下配置，tickLine。
